enterprise/micronaut/src/org/netbeans/modules/micronaut/MicronautConfigProperties.java

```diff
@@ -60,6 +94,16 @@ public static boolean hasConfigMetadata(Project project) {
                     Exceptions.printStackTrace(ex);
```

Comment:
shouldn't exception's propagete instead of being swallowed?

Reply:
Why? This way, the exception is printed to log, user gets immediately notified that something was wrong at the particular place, while the upper level functionality could stay (at least partially) working.